pyk4a/calibration.py

```diff
@@ -0,0 +1,79 @@
+from enum import IntEnum
+from typing import List, Tuple
+import k4a_module
+from .pyk4a import Result, K4AException, K4ATimeoutException, PyK4A
+from pyk4a.config import Config
+
+
+class CalibrationType(IntEnum):
+    UNKNOWN = -1       # Unknown
+    DEPTH = 0          # Depth Camera
+    COLOR = 1          # Color Sensor
+    GYRO = 2           # Gyroscope
+    ACCEL = 3          # Accelerometer
+    NUM = 4            # Number of types excluding unknown type
+
+
+class Calibration:
+    def __init__(self, device: PyK4A, config: Config):
+        if isinstance(device, PyK4A):
+            self.device = device
+        else:
+            raise K4AException(f'Calibration instance created without '
+                               f'a device of the proper class')
+        self.config = config
+
+    def convert_3d_to_3d(self,
+                         source_point_3d: List,
+                         source_camera: CalibrationType,
```

Comment:
Why not set the source and target in the constructor ?

Reply:
I like the idea. My thought was there be cases where you would want different point of views for different transformations? Maybe in this case we could  use 2 Calibrations...? thoughts?